_drafts/release-workflows.md

```diff
@@ -0,0 +1,309 @@
+
+- LTS branches and distribution channels
+- Future branches and distribution channels
+- Pre-releases
+
+## Proposed solution
+
+Support the following type of branches:
+- [Single release branch](#single-release-branch)
+- [Non release branch](#non-release-branch)
+- [Future branches](#future-branches)
+- [LTS branches](#lts-branches)
+- [Pre-releases](#pre-releases)
+
+## Detailed design
+
+**Terminology:**
+
+| Term                | Definition                                                                                                                                                                         |
+|---------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Default branch      | Git branch that contains the stable code (made available to all users). Usually named `master`.                                                                                    |
+| Future branches     | Permanent Git branch that contains the unstable code (made available only to early adopters). Usually named `next`.                                                                |
+| Channel             | Distribution channels used to limit the accessibility of a release to a subset of users. For example `dist-tags` in `npm`.                                                         |
+| Default channel     | Distribution channel to make stable version available to every users. Usually named `latest`.                                                                                      |
+| Future channels     | Distribution channels to make unstable version available to early adopters. Usually named `next`.                                                                                  |
+| Pre-release version | version that doesn't follow the regular `semver` increment pattern. For example any increment of `2.0.0-beta.1` is `2.0.0-beta.2`, then `2.0.0-beta.2` etc...                      |
+| Version             | version following the `semver` rules. For example, a `minor` increment of `1.0.0` is `1.1.0`.                                                                                      |
+| Module              | A software meant to be used as a dependency of other software. It's not meant to be used by final users.                                                                           |
+| Application         | A software meant to be used by final users. No other software depends on it.                                                                                                       |                                                         |
+
+### Branch types
+
+#### Single release branch
+
+Simple workflow publishing release for each commits on the `master` branch.
+
+Ideal for small or stable modules receiving few breaking changes.
+
+Branch: `master`
```

Comment:
Instead of saying `master`, perhaps just say `The _default_ branch` to emphasize that `semantic-release` simply uses the branch that is configured as the `default` branch on GitHub (or the user's SCM platform).

Reply:
See https://github.com/semantic-release/evolution/pull/1#discussion_r194782438